packages/plugins/Gfi/src/store/getInitialState.ts

```diff
@@ -3,6 +3,7 @@ import { GfiState } from '../types'
 const getInitialState = (): GfiState => ({
   featureInformation: {},
   visibleWindowFeatureIndex: 0,
+  visibilityChangeIndicator: 0,
```

Comment:
Missing changelog

Reply:
This is an internal solution belonging to the list development. I don't think it needs a changelog entry, unless it was a bugfix on an existing release.